man/systemd.network.xml

```diff
@@ -688,6 +688,56 @@
             which is then configured to use them explicitly.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Home=</varname></term>
+          <listitem>
+            <para>Designates this address the "home address" as defined in
+            <ulink url="https://tools.ietf.org/html/rfc6275">RFC 6275</ulink>.
+            Works only for IPv6.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>NoDAD=</varname></term>
+          <listitem>
+            <para>Do not perform Duplicate Address Detection
+            <ulink url="https://tools.ietf.org/html/rfc4862">RFC 4862</ulink> when adding this address.
+            Works only for IPv6.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>ManageTempAddr=</varname></term>
```

Comment:
ManageTemporaryAddress=.

Please state what the argument must be, and if the default unset means disabled.